common/predictive-text/package-lock.json

```diff
@@ -6,7 +6,7 @@
 	"packages": {
 		"": {
 			"name": "@keymanapp/lexical-model-layer",
-			"version": "15.0.80",
+			"version": "15.0.94",
```

Comment:
Can we remove this from the PR given version bumps are the responsibility of the auto increment script?

Reply:
I thought I rebased it out.  You betcha.